assets/misc/nmdc_schema_temp_checker.py

```diff
@@ -0,0 +1,87 @@
+from linkml_runtime import SchemaView
+from src.scripts.make_typecode_to_class_map import get_collection_names_for_class_name
+from nmdc_api_utilities.collection_search import CollectionSearch
+import pandas as pd
+
+schema_url = "https://raw.githubusercontent.com/microbiomedata/nmdc-schema/refs/heads/main/nmdc_schema/nmdc_materialized_patterns.yaml"
```

Comment:
this is fine for now, but it's better to get the schema from the repo package fiels, not over HTTP

Reply:
There are some illustrations of that in https://github.com/microbiomedata/nmdc-schema/blob/main/tests/test_nmdc_data.py. There are other ways to do it too.

Reply:
I would like to hear more about

  • Downgrading rdflib (6.3.2 -> 6.2.0)
  • Installing matplotlib (3.10.3)
  • Installing nmdc-api-utilities (0.3.9)